values.yaml

```diff
@@ -56,7 +56,7 @@ global:
   blackboxExporterEnabled: false
 
   # Used to enable Celery autoscaling
-  kedaEnabled: false
+  kedaEnabled: true
```

Comment:
I'm not sure this should change... @sjmiller609 what do you think?

Reply:
@danielhoherd fixed

Reply:
This was not fixed. I'd like additional opinions on changing this default value. @schnie ?

Reply:
Oh wait, my bad. This was fixed by moving it to a different file!